journal-temp/modernized/16-chapter-16.adoc

```diff
@@ -727,7 +770,8 @@ to the substance and life of Christianity, to true love, fervent charity,
 and tender-heartedness, and forgiveness towards one another,
 and to follow peace with all men, and holiness; without which no man shall see the Lord.
 
-A Postscript, relating to the doctrine of the Resurrection and Eternal Judgment.
+[.blurb]
```

Comment:
This didn't seem like a true postscript to me, so I just put it as a blurb...but you may wanna change it.